plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongo3LatestConnectorSmokeTest.java

```diff
@@ -18,8 +18,8 @@
 
 import static io.trino.plugin.mongodb.MongoQueryRunner.createMongoQueryRunner;
 
-public class TestMongo3DistributedQueriesLatest
-        extends BaseMongoDistributedQueries
+public class TestMongo3LatestConnectorSmokeTest
```

Comment:
latest are not latest any more. And honestly I do not get why these specific versions are tested :)

Reply:
Let's consider this out of scope here and see https://github.com/trinodb/trino/issues/5804

see also my reply in https://github.com/trinodb/trino/pull/7179#discussion_r589292509